<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1083</width>
    <height>745</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 700 14pt &quot;Sans&quot;;
background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>120</y>
      <width>481</width>
      <height>431</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(205, 224, 247);
color:rgb(0, 0, 0);
border-radius:rgb(0, 0, 0);
    border-radius: 20px;</string>
    </property>
    <property name="title">
     <string>                                   Color Changer </string>
    </property>
    <widget class="QPushButton" name="buttonRouge">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>330</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: rgb(224, 27, 36);
    color: white;
    border: none;
    width: 50px;
    height: 50px;
    border-radius: 20px;
}

QPushButton:hover {
    background-color: rgb(224, 27, 36);
}

QPushButton:pressed {
    background-color: rgb(165, 29, 45);
}
</string>
     </property>
     <property name="text">
      <string>R</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonBleu">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>330</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: rgb(28, 113, 216);
    color: white;
    border: none;
    width: 50px;
    height: 50px;
    border-radius: 20px;
}
 </string>
     </property>
     <property name="text">
      <string>B</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>70</y>
       <width>231</width>
       <height>221</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
         background-color: rgb(255, 255, 255);
         color: rgb(0, 0, 0);
          padding: 10px;
         border: 2px solid #2980b9;
        
        }</string>
     </property>
     <property name="text">
      <string>             COLOR</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonGreen">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>330</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color:  rgb(46, 194, 126);
    color: white;
    border: none;
    width: 50px;
    height: 50px;
    border-radius: 20px;
}</string>
     </property>
     <property name="text">
      <string>G</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonSurprise">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>330</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: rgb(255, 255, 255);
    color:  rgb(0, 0, 0);
    border: none;
    width: 50px;
    height: 50px;
    border-radius: 20px;
}</string>
     </property>
     <property name="text">
      <string>Suprise Me</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonYellow">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>330</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color:  rgb(246, 211, 45);
    color: white;
    border: none;
    width: 50px;
    height: 50px;
    border-radius: 20px;
}</string>
     </property>
     <property name="text">
      <string>Y</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonReset">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>390</y>
       <width>81</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: rgb(255, 255, 255);
    color:  rgb(0, 0, 0);
    border: none;
    width: 50px;
    height: 50px;
    border-radius: 9px;
}</string>
     </property>
     <property name="text">
      <string>Reset</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>QPushButton {</normaloff>QPushButton {</iconset>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1083</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
